.gitlab-ci.yml

```diff
@@ -286,9 +304,27 @@ linux32-test:
   variables:
     BUILD_TARGET: linux32
 
+linux32_undef-test:
+  extends:
+    - .test-template
+    - .skip-in-fast-mode-template
+  needs:
+    - linux32_undef-build
+  variables:
+    BUILD_TARGET: linux32_undef
+
 linux64-test:
   extends: .test-template
   needs:
     - linux64-build
   variables:
     BUILD_TARGET: linux64
+
+linux64_thread-test:
+  extends:
+    - .test-template
+    - .skip-in-fast-mode-template
+  needs:
+    - linux64_thread-build
+  variables:
+    BUILD_TARGET: linux64_thread
```

Comment:
```suggestion
    BUILD_TARGET: linux64_thread

```

Reply:
Fixed it. One question regarding the namings: would it be better to name the targets tsan/ubsan instead of thread/undef in order to be consistent with the sanitizer's official naming?

Reply:
Yeah, maybe. Thoughts @PastaPastaPasta ?

Reply:
I switched to tsan/ubsan. If @PastaPastaPasta prefers the other namings, I will switch back.